webdriver-spec.html

```diff
@@ -2022,6 +2022,9 @@
    <li><p>Let <var>value</var> be the result of <a>getting a
     property</a> named <var>name</var> from <var>parameter</var>.
 
+   <li><p>Let <var>key</var> be the result of
+    lower-casing <var>key</var>.
```

Comment:
`<a>ASCII lower alpha</a>`, which is from the Infra spec: https://infra.spec.whatwg.org/#ascii-lower-alpha